test.py

```diff
@@ -40,665 +41,601 @@
 			)
 		},
 	template_places = {
-		# *-* CANYON *-*
-		Place(
-			"in un gigantesco canyon di terra rossa che si estende per chilometri; da qui forse riusciresti a scalarne i muri",
-			exits={
-				Exit('verso la cima, scalando le rocce', {'bordo'}),
-				Exit('lungo il canyon', {'canyon'})
+		"Canyon": {
+			"chances": 5,
+			"descriptions": [
+				"Un enorme canyon di terra rossa",
+				"Un piccolo canyon"
```

Comment:
Queste dovrebbero iniziare per minuscola ed essere il continuo di "sei...", quindi contenendo "in"
(anche tutte le altre)